src/masternodeman.cpp

```diff
@@ -444,24 +444,22 @@ void CMasternodeMan::CheckSpentCollaterals(const std::vector<CTransactionRef>& v
     }
 }
 
-static void addMnToLastPaid(const CMasternodeMan* manager,
-                            std::vector<std::pair<int64_t, MasternodeRef>>& vecMasternodeLastPaid,
-                            bool fFilterSigTime, const MasternodeRef& mn, int minProtocol, int nMnCount,
-                            int nBlockHeight, const CBlockIndex* pBlockReading)
+static bool canScheduleMN(bool fFilterSigTime, const MasternodeRef& mn, int minProtocol, int nMnCount,
+                          int nBlockHeight, const CBlockIndex* pBlockReading)
```

Comment:
Can remove `pBlockReading` argument, it's not used anymore.
And would be good to squash this commit into 89416a1 that introduced the `addMnToLastPaid` function.

Reply:
Yeah, the reason why I didn't squash, is to let you review faster, without having to go over the previous commits again.
Will squash now.

Reply:
kk, all good.